department-of-veterans-affairs/caseflow
# 1521: make distribution tasks visible to VSOs, plus a spec.

Resolves #{1521}

### Description
This only adds distribution tasks to the case timeline for VSOs.  This wasn't the original plan for this ticket, but it seems to be what it has evolved into.

### Acceptance Criteria
- [x] Rspec test

### Testing Plan
1. create an appeal like this:
```ruby
RequestStore[:current_user] = User.system_user
appeal = FactoryBot.create(:appeal, :with_ihp_task, :dispatched, :with_decision_issue,
               docket_type: "direct_review",
               stream_type: "original")
appeal.uuid
```
2. check it out at http://localhost:3000/queue/appeals/:uuid as both a VSO user, and another user such as a COB.
3. you should see the expected timeline events present: a NOD, dispatched, an IHP task, and a distribution task.
